workflows/microbiome/taxonomic-binninge-evaluation/FAIRyMAGs-taxonomic-binning-evaluation.ga

```diff
@@ -0,0 +1,8824 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "This Workflow task is to use read(s) (in a paired list) and the assembly(s) to start binning them with multiple different binners. After they get taxonomic classified with GTDB-Tk. With the Gold standard Binning file (gsb) CAMI Amber can evaluate the result of each binner. In addition to this CAMI Amber also provides a taxonomic classification evaluation. Both evaluations come with certain TSV file(s) and an HTML report for visualization of the result in plots or as tables. The main metric to check how good a binner was area provided in multiple ways and when using multiple samples it is also possible to check the result for both evaluation per sample and all together. ",
```

Comment:
Maybe something like
> Multi-algorithm metagenomic binning evaluation pipeline with taxonomic classification and CAMI AMBER benchmarking.
Processes assemblies and reads through CONCOCT, MetaBAT2, SemiBin, and MaxBin2 binning algorithms, followed by DAS Tool/Binette optimization. Performs GTDB-Tk taxonomic classification with NCBI mapping, then evaluates all results against gold standard using CAMI AMBER framework. Generates comprehensive HTML reports, performance metrics, and comparison visualizations for systematic binning algorithm assessment.